ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -63,7 +63,7 @@ Listen <%= addr_port %>
   <%- end -%>
 
   RewriteEngine On
-  RewriteRule ^(.*) <%= @ssl ? "https" : "http" %>://<%= @proxy_server %>:<%= @port %>$1 [R=301,NE,L]
+  RewriteRule ^(.*) <%= @ssl ? "https" : "http" %>://%{HTTP_HOST}:<%= @port %>$1 [R=301,NE,L]
```

Comment:
I think this change will work to fix #3442 so that any http -> https redirects will respond to the various server aliases we have.  I should actually hop on dev to confirm this.

Reply:
I can confirm this works in dev, but could still use a 2nd pair of eyes.

Reply:
This looks correct.  I think this will even work with Proxies since HTTP_HOST would be the address used to access OnDemand.